public/javascripts/cohortVizController.js

```diff
@@ -38,6 +40,16 @@ var cohortVizController = function($scope, $http, $state) {
 
     var starsList = [];
     $scope.starData = formatPieData($scope.stars, starsList);
+
+    var engagePeerList = [];
+    $scope.engagePeerData = formatPieData($scope.engagePeer, engagePeerList);
+
+    var engageAdultList = [];
+    $scope.engageAdultData = formatPieData($scope.engageAdult, engageAdultList);
+
+    var engageContentList = [];
+    $scope.engageContentData = formatPieData($scope.engageContent, engageContentList);
+   
```

Comment:
I think these lines got changes somewhere else, but idk? Passing the empty list is unneccessary. I think is was changed on cohort-timefilter